CHANGES.md

```diff
@@ -22,6 +22,7 @@ We adhere to the [keepachangelog](https://keepachangelog.com/en/1.0.0/) format (
 * Bump default `diktat` version `1.2.4.2` -> `1.2.5`. ([#1631](https://github.com/diffplug/spotless/pull/1631))
 * Bump default `flexmark` version `0.62.2` -> `0.64.0`. ([#1302](https://github.com/diffplug/spotless/pull/1302))
 * Bump default `googleJavaFormat` version `1.15.0` -> `1.16.0`. ([#1630](https://github.com/diffplug/spotless/pull/1630))
+* Support Rome as a formatter for JavaScript and TypeScript code. Adds a new `rome` step to `javascript` and `typescript` formatter configurations. https://github.com/diffplug/spotless/pull/1663
```

Comment:
This should go under the `### Added` section of `## Unreleased`

Reply:
Closer! You moved the entry to be under `## [2.38.0] - 2023-04-06`, but it needs to be under `## Unreleased`, and above `## [2.38.0]`. This PR hasn't been released yet, so it should only be in the unreleased section :)